code/modules/client/preference/character.dm

```diff
@@ -551,7 +556,7 @@
 	loadout_gear = sanitize_json(loadout_gear)
 	custom_emotes_tmp = sanitize_json(custom_emotes_tmp)
 	custom_emotes = init_custom_emotes(custom_emotes_tmp)
-
+	cyborg_brain_type = sanitize_text(cyborg_brain_type, initial(cyborg_brain_type))
```

Comment:
This should be a sanitize inlist so that it fails if the type from the DB isn't valid.